checklist_plus/text_generation/llm.py

```
@@ -0,0 +1,391 @@
+                        filtered_paraphrases.append(filtered_paraphrases[0])
+
+                all_paraphrases.extend(filtered_paraphrases[:n_paraphrases])
+
+        except Exception as e:
+            logger.error("LLM batch paraphrase failed", exc_info=True)
+            raise
+
+        return all_paraphrases
+
+    def filter_options(self, texts, word, options, threshold=5, **kwargs):
+        if type(texts) != list:
```

Ruff (0.12.2)

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)